spec/integration/knife/config_use_spec.rb

```diff
@@ -67,11 +67,6 @@
       ENV["HOME"] = path_to(".")
     end
 
-    context "with no argument" do
-      subject { knife_use_profile.stderr }
-      it { is_expected.to eq "FATAL: You must specify a profile\n" }
-    end
```

Comment:
This test was failing because `knife config use` with no argument is now a valid command.